alienfile

```diff
@@ -16,11 +16,12 @@ configure {
 };
 
 plugin 'Probe::CBuilder' => (
-  cflags  => Alien::OpenMP::configure->cflags,
-  lang    => 'C',
-  libs    => Alien::OpenMP::configure->libs,
-  options => {quiet => 0},
-  program => join("\n" => <DATA>),
+  lang         => 'C',
+  cflags       => Alien::OpenMP::configure->cflags,
+  libs         => Alien::OpenMP::configure->libs,
+  auto_include => Alien::OpenMP::configure->auto_include,
```

Comment:
`auto_include` is not an attribute/property for `CBuilder` plugin.

To do, what I believe needs to happen, is augment the after `probe` block to be like and add an `auto_include` method to `Alien::OpenMP` (see below).
```perl
# only reached on success AFAICT
after probe => sub {
  my $build = shift;
  $build->install_prop->{'alien_openmp_compiler_has_openmp'} = 1;
  $build->runtime_prop->{auto_include}                       = Alien::OpenMP::configure->auto_include;
};
```

Reply:
mind a suggestion for this, I like how GH allows for this

Reply:
I think the lines have to all be in the diff to allow for a suggestion to apply correctly.